parquet-common/src/main/java/org/apache/parquet/bytes/ByteBufferInputStream.java

```diff
@@ -28,6 +28,16 @@
 
 import org.apache.parquet.ShouldNeverHappenException;
 
+/*
+Changes implemented:
```

Comment:
Can you remove this? Generally, we want PRs that can be merged. You can add comments about what is changing in the PR rather than in the code.

Reply:
Removed